recordset/index.html.in

```diff
@@ -32,6 +32,7 @@
                             ng-href="{{ edit() }}" tooltip-placement="bottom" uib-tooltip="{{ ((vm.pageLimit > vm.RECORDEDIT_MAX_ROWS) ? 'Editing disabled when items per page > ' + vm.RECORDEDIT_MAX_ROWS : 'Click here to edit this page of records.' ) }}">
                             <span class="glyphicon glyphicon-pencil"></span> <strong>Edit</strong>
                         </a>&nbsp;&nbsp;
+                        <export ng-if="ctrl.showExportButton" has-values="vm.rowValues.length>0" reference="vm.reference"></export>&nbsp;&nbsp;
```

Comment:
The 2 `&nbsp;`s are going to be drawn regardless of this button being shown or not. Maybe have an outer `div` with the `ng-if` that contains the `<export>` div and the 2 `&nbsp;`s